docker/router-prod.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
```

Comment:
Not strictly necessary but if I were you I would split my docker compose into one common one then another one that has only the additions for prod. Then you run it with multiple composes that makes it less likely to add a config to dev and then forget to add it to prod.